whois-nrtm4/src/main/java/net/ripe/db/nrtm4/persist/NrtmDocumentType.java

```diff
@@ -0,0 +1,10 @@
+package net.ripe.db.nrtm4.persist;
+
+public enum NrtmDocumentType {
+    DELTA,
+    SNAPSHOT;
+
+    public String nameToLowerCase() {
+        return name().toLowerCase();
```

Comment:
If we need the value in lowercase always, we should override toString method, so whenever we call just DELTA , it will automatically returns me the lowercase instead of remembering to call nameToLowerCase. But feel free to ignore my comment.

Reply:
no, that's a good idea. i'll do that.